Dockerfile

```
@@ -0,0 +1,8 @@
+FROM node:16-alpine
+WORKDIR /var/www
+COPY "package.json" .
```

package-lock тоже стоит копировать
```suggestion
COPY package*.json .
```